docs/en/latest/practices/step-by-step.md

```diff
@@ -408,7 +430,7 @@ It will output like:
 
 ## Install APISIX Ingress Controller
 
-APISIX ingress controller can help you manage your configurations declaratively by using Kubernetes resources.
+APISIX ingress controller can help you manage your configurations declaratively by using Kubernetes resources. Here we will install version 0.5.0.
```

Comment:
Now the latest version of ingress controller is 0.6.0.

Reply:
Yamls picked from helm-charts. It uses 0.5.0

Reply:
cc @tokers need to update the helm repo.